deflate_stored.c

```diff
@@ -132,7 +132,7 @@ Z_INTERNAL block_state deflate_stored(deflate_state *s, int flush) {
         }
         s->block_start = (int)s->strstart;
     }
-    s->high_water = MIN(s->high_water, s->strstart);
+    s->high_water = MAX(s->high_water, s->strstart);
```

Comment:
These were incorrectly converted to `MIN` in https://github.com/zlib-ng/zlib-ng/commit/d4dd539d1c08ce0affb360120a9e4ac2396e734b. Now have been corrected.